src/app/storify/game/game.component.html

```diff
@@ -1,13 +1,19 @@
 <app-loader *ngIf="loading"></app-loader>
 <app-submission
-  *ngIf="gameState === 'submitting' && !loading"
+  *ngIf="gameState === 1 && !loading"
   [lobby]="lobby"
   [story]="story"
   (submit)="submitSentence($event)"
   (tick)="tick($event)"
 ></app-submission>
+<app-submissions-view
+  *ngIf="(gameState === 2 || gameState === 3) && !loading && !submissionsViewed"
```

Comment:
can you make a comment about the state or use enums or something, this is very not readable